CODEOWNERS

```diff
@@ -0,0 +1,91 @@
+# Catch-all
+* @cilium/janitors
+
+# BPF Data Path
+bpf/ @cilium/bpf
+pkg/maps/ @cilium/bpf
+pkg/bpf/ @cilium/bpf
+pkg/policy/prefilter.go @cilium/bpf
+daemon/prefilter.go @cilium/bpf
+
+
+# Agent -> Kernel interaction
+# TODO create a separate group consisting of @scanf and @borkmann
+bpf/*.sh @cilium/bpf
+
+# CI
+tests/ @cilium/ci
+Jenkinsfile @cilium/ci
+
+# KVStore
+pkg/kvstore/ @cilium/kvstore
+
+# K8s integration, K8s CNI plugin
+daemon/k8s_watcher.go @cilium/kubernetes
+pkg/k8s/ @cilium/kubernetes
+plugins/cilium-cni/ @cilium/kubernetes
+examples/kubernetes/ @cilium/kubernetes
+examples/minikube/ @cilium/kubernetes
+
+# Docker plugin
+plugins/cilium-docker/ @cilium/docker
+
+#TODO: If mesos plugin is created: @nebril @manalibhutiyani @aanm
+
+# Policy behaviour
+pkg/policy @cilium/policy
+
+# Packaging
+contrib/packaging/deb/ @eloycoto
+contrib/packaging/docker/ @aanm @ianvernon
+
+# Proxy
+pkg/proxy/ @cilium/proxy
+
+# based on not yet merged PR 1580
+pkg/envoy/ @cilium/proxy
+envoy/ @cilium/proxy
+envoy-api/ @cilium/proxy
+# TODO: where is kproxy located? @jrfastab @manalibhutiyani
+pkg/proxy/oxyproxy.go @tgraf
+
+# Agent API
+api/ @cilium/api
+pkg/policy/api/ @cilium/api
+pkg/apierror/ @cilium/api
+pkg/apisocket/ @cilium/api
+pkg/proxy/accesslog @cilium/api
+monitor/payload @cilium/api
+
+# CLI
+cilium/ @cilium/cli
+
+# Docs
+Documentation/ @cilium/docs
+
+# BPF Docs
+Documentation/bpf.rst @scanf @borkmann
+
+# Developer docs
+Documentation/contributing.rst @cilium/contributing
+
+# Examples
+Examples/ @cilium/docs
+
+# Dependencies/vendor
+vendor/ @cilium/vendor
+
+# Monitor
+monitor/ @cilium/monitor
+
+# Agent
+daemon/ @cilium/agent
+
+# TODO: Endpoint generation - where is it? @manalibhutiyani @andre @joestringer
```

Comment:
This is difficult to create an 'owner' of because endpoint regeneration is triggered by the daemon based off of various events occurring. I suggest we get rid of this 'endpoint generation' owner and make it owned by those who manage `daemon/` and `endpoint/`. @tgraf what do you think?

Reply:
I think we should isolate this code eventually. We can defer this dedicated owner until that has happened.